src/pages/introduce/introduce.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
+import "../../style/common.scss";
+import "./introduce.scss";
+
+const Profile = () => {
+  const navigate = useNavigate();
+  const ToProduct = () => {
+    navigate("/product");
+  };
+  function handleInput() {
+    console.log("input 태그 동작");
+  }
+
+  return (
+    <div className="container">
```

Comment:
```suggestion
    <div className="profile">
```
scss를 사용할 때 가장 부모가 되는 태그의 className을 
컴포넌트의 이름과 동일하게 하는 것이 컨벤션입니다